contrib/vagrant/deployments/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "http-tcp"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   ["cilium.io"]
+        apiVersions: ["v2"]
+        operations:  ["CREATE", "UPDATE"]
+        resources:   ["ciliumnetworkpolicies"]
+  validations:
+    - expression:
+        (has( object.spec.ingress[0].toPorts[0].rules.http) &&
```

Comment:
This is only testing the first element of the ingress or egress section. Shouldn't we test all elements?